lib/cmd_calc.go

```diff
@@ -0,0 +1,366 @@
+package lib
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/spf13/pflag"
+	"math"
+	"math/big"
+	"net"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// CmdCalcFlags are flags expected by CmdCalc
+type CmdCalcFlags struct {
+	Help    bool
+	NoColor bool
+}
+
+// Init initializes the common flags available to CmdCalc with sensible
+func (f *CmdCalcFlags) Init() {
+	_h := "see description in --help"
+	pflag.BoolVarP(
+		&f.Help,
+		"help", "h", false,
+		"show help.",
+	)
+	pflag.BoolVar(
+		&f.NoColor,
+		"nocolor", false,
+		_h,
+	)
+}
+
+// Stack type
+type Stack []string
```

Comment:
We already have stack implementation in Golang. We can use that instead of implementing our own.
Does the Golang stack have some limitations to not using it?